Reactor/Utilities/InnerNetObjectManager.cs

```
@@ -0,0 +1,91 @@
+    /// </summary>
+    /// <typeparam name="T">The type of the <see cref="InnerNetObject"/> to spawn. Must inherit from <see cref="InnerNetObject"/>.</typeparam>
+    /// <param name="ownerId">The owner ID for the spawned object. Defaults to -2, which typically means no specific owner.</param>
+    /// <param name="spawnFlags">The spawn flags to use when spawning the object. Defaults to <see cref="SpawnFlags.None"/>.</param>
+    /// <returns>The newly spawned <see cref="InnerNetObject"/>, if prefab is null then it will return null.</returns>
+    public static InnerNetObject? SpawnNewNetObject<T>(int ownerId = -2, SpawnFlags spawnFlags = SpawnFlags.None) where T : InnerNetObject
+    {
+        if (!AmongUsClient.Instance.AmHost)
+        {
+            Warning("You can only spawn a InnerNetObject as Host.");
+            return null;
+        }
+
+        var netObj = GetNetObjPrefab<T>();
+
+        if (netObj == null)
+        {
+            return null;
+        }
+
+        var netObjSpawn = UnityEngine.Object.Instantiate(netObj);
+        AmongUsClient.Instance.Spawn(netObjSpawn, ownerId, spawnFlags);
+        return netObjSpawn;
+    }
+
+    /// <summary>
+    /// Spawns an existing <see cref="InnerNetObject"/> instance on the network.
+    /// </summary>
+    /// <param name="netObj">The <see cref="InnerNetObject"/> instance to spawn.</param>
+    /// <param name="ownerId">The owner ID for the spawned object. Defaults to -2, which typically means no specific owner.</param>
+    /// <param name="spawnFlags">The spawn flags to use when spawning the object. Defaults to <see cref="SpawnFlags.None"/>.</param>
+    public static void SpawnNetObject(this InnerNetObject netObj, int ownerId = -2, SpawnFlags spawnFlags = SpawnFlags.None)
+    {
+        if (!AmongUsClient.Instance.AmHost)
+        {
+            Warning("You can only spawn a InnerNetObject as Host.");
```

Why the decision to squash this error here? TOCTOU concerns?

InnerNetClient logs an error when AUClient.Instance.Spawn is called on clients

---

Thought it would be better to know that the warning is directly coming from reactor